modules/snooker/src/main/ts/ephox/snooker/api/CopySelected.ts

```diff
@@ -1,22 +1,28 @@
 import { Arr, Obj } from '@ephox/katamari';
-import { Attribute, Css, Insert, Remove, Selectors, SugarElement } from '@ephox/sugar';
+import { Attribute, Insert, Remove, Replication, Selectors, SugarElement, Width } from '@ephox/sugar';
+import * as ColUtils from '../util/ColUtils';
```

Comment:
This is going to fail once the new linter rules are merged, as it's out of order. It might be good to move it above `LayerSelector` so it's in the right place?